qiita_pet/support_files/doc/source/tutorials/getting-started.rst

```diff
@@ -1,328 +0,0 @@
-.. _getting-started:
-
-.. index:: getting-started
-
-Getting started
-===============
-
-
-.. note::
-   This is a work in progress. Please let us know of any additions or changes
-   you will like to see by emailing us at `qiita.help@gmail.com
-   <qiita.help@gmail.com>`__.
-
-If you have not yet created an account, please see the document
-:doc:`account-creation`.
-
-
-Managing your studies
----------------------
-
-Studies are the main source of data for Qiita. Studies can contain only one set
-of samples but can contain multiple sets of raw data, each of which can have a
-different preparation. Many experiments will contain only one data set, which
-includes data for all samples. Such experiments will include a single sample
-template and a single prep template.  
-
-However, Qiita can also support more complex study designs. For example
-imagine a study with 100 samples in which:
```

Comment:
CMI workshop does not cover those complex designs so far. But there is meta-analysis section (http://cmi-workshop.readthedocs.io/en/latest/qiita-16S-analysis.html#creating-a-meta-analysis).